sdk-core/src/main/java/io/milvus/v2/service/collection/request/AddCollectionFieldReq.java

```diff
@@ -19,12 +19,203 @@
 
 package io.milvus.v2.service.collection.request;
 
-import lombok.Data;
-import lombok.experimental.SuperBuilder;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
 
-@Data
-@SuperBuilder
-public class AddCollectionFieldReq extends AddFieldReq{
+public class AddCollectionFieldReq extends AddFieldReq {
     private String collectionName;
     private String databaseName;
+
+    private AddCollectionFieldReq(Builder builder) {
+        super(builder);
+        this.collectionName = builder.collectionName;
+        this.databaseName = builder.databaseName;
+    }
+
+    public String getCollectionName() {
+        return collectionName;
+    }
+
+    public void setCollectionName(String collectionName) {
+        this.collectionName = collectionName;
+    }
+
+    public String getDatabaseName() {
+        return databaseName;
+    }
+
+    public void setDatabaseName(String databaseName) {
+        this.databaseName = databaseName;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Is this method necessary? We don't have special comparison logic for request classes.